src/components/Accordion/Accordion.stories.tsx

```diff
@@ -36,9 +37,12 @@ export default {
       ),
     },
   },
+  args: {
+    variant: AccordionIconVariant.Primary,
+  },
 } as ComponentMeta<typeof Accordion>;
 
-const Template: ComponentStory<typeof Accordion> = () => {
+const Template: ComponentStory<typeof Accordion> = (args) => {
```

Comment:
`args` can be removed

Reply:
How can I then keep this functionality in storybook?
<img width="1634" alt="image" src="https://user-images.githubusercontent.com/31471142/188815740-51083326-7ab7-47cb-8986-fa7c4405a4dd.png">

Reply:
When I made this comment, `args` was not in use. That is not the case anymore, so now it of course has to be there.